<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>App</class>
 <widget class="QWidget" name="App">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>529</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(29, 46, 62);
}
QPushButton{
	padding: 10px 10px;
	border:none;
	color:white;
}
QPushButton:hover{
	background-color: rgb(45, 66, 75);
}
QPushButton:focus{
	background-color:#00CCDD;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0,0,0">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnAccueil">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Accueil</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnLivre">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Livres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnMembre">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Membres</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnEmprunt">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Emprunts</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="pageAccueil"/>
     <widget class="QWidget" name="pageLivre">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QFrame" name="frame_7">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QLineEdit" name="lineEditRecherche">
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	border: 1px solid white;
	border-radius: 5px;
}

QLineEdit:focus{
	border: 1px solid black;
}</string>
            </property>
            <property name="placeholderText">
             <string>Recherche</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnRecherche">
            <property name="font">
             <font>
              <bold>false</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <property name="mouseTracking">
             <bool>true</bool>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::EditFind"/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_8">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Trier par armoire</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxChoixArmoir">
            <item>
             <property name="text">
              <string>Tous</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>A</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>B</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>C</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>D</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Trier par genre :</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBoxChoixGenre"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="tableWidget">
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
         </property>
         <column>
          <property name="text">
           <string>Titre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Genre</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Auteur</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Maison d'édition</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Propriétés</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Quantité</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Armoire</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="frame_2">
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QPushButton" name="btnAjouterLivre">
            <property name="text">
             <string>Nouveau</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnModifierLivre">
            <property name="text">
             <string>Modifier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnSupprimerLivre">
            <property name="text">
             <string>Supprimer</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageMembre">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QFrame" name="frame_5">
         <property name="styleSheet">
          <string notr="true">QFrame{
	background-color: rgb(29, 46, 62);
}
QPushButton{
	padding: 10px 10px;
	border:none;
	color:white;
}
QPushButton:hover{
	background-color: rgb(45, 66, 75);
}
QPushButton:focus{
	background-color:#00CCDD;
}</string>
         </property>
         <property name="frameShape">
          <enum>QFrame::Shape::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Shadow::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QPushButton" name="btnMembreEtudiant">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Etudiants</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnMembreAdulte">
            <property name="font">
             <font>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Adultes</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QStackedWidget" name="stackedWidget_2">
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="pageEtudiant">
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QTableWidget" name="tableWidget_2">
             <column>
              <property name="text">
               <string>Nom</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prénom(s)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de naissance</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Niveau d'étude</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Sexe</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Adresse</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Numéro de téléphone</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_4">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QPushButton" name="btnAjouterEtudiant">
                <property name="text">
                 <string>Nouveau</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnModifierEtudiant">
                <property name="text">
                 <string>Modifier</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnSupprimerEtudiant">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="pageAdulte">
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QTableWidget" name="tableWidget_3">
             <column>
              <property name="text">
               <string>Nom</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Prénom(s)</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date de naissance</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Fonction</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Sexe</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Adresse</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Numéro de téléphone</string>
              </property>
             </column>
            </widget>
           </item>
           <item>
            <widget class="QFrame" name="frame_6">
             <property name="frameShape">
              <enum>QFrame::Shape::StyledPanel</enum>
             </property>
             <property name="frameShadow">
              <enum>QFrame::Shadow::Raised</enum>
             </property>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QPushButton" name="btnAjouterAdulte">
                <property name="text">
                 <string>Nouveau</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnModifierAdulte">
                <property name="text">
                 <string>Modifier</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="btnSupprimerAdulte">
                <property name="text">
                 <string>Supprimer</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Orientation::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pageEmprunt"/>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="styleSheet">
      <string notr="true">QFrame{
	background-color: rgb(29, 46, 62);
}
QPushButton{
	padding: 10px 10px;
	border:none;
	color:white;
}
QPushButton:hover{
	background-color: rgb(45, 66, 75);
}
QPushButton:focus{
	background-color:#00CCDD;
}</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnParametre">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Parametre</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnDeconnexion">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Déconnexion</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
